_re2.cc

```diff
@@ -65,6 +61,43 @@ typedef struct _MatchObject2 {
   StringPiece* groups;
 } MatchObject2;
 
+// getters for MatchObject2
+static PyObject* match_pos_get(MatchObject2* self)
```

Comment:
Why not use the proper getter signature here and just cast to MatchObject2 internally?

Reply:
Silly question but when you say "proper getter signature", what do you mean?

I had a look at other python extensions and the python documentation before implementing the getters and they both suggest using this kind of signatures. Eg: https://docs.python.org/2.7/extending/newtypes.html